src/modules/jcms_notifications/jcms_notifications.module

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Contains jcms_notifications.module..
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function jcms_notifications_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the jcms_notifications module.
+    case 'help.page.jcms_notifications':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Integrates with Amazon SQS and Amazon SNS.') . '</p>';
+      return $output;
+
+    default:
+  }
+}
+
+/**
+ * Implements hook_cron().
+ */
+function jcms_notifications_cron() {
```

Comment:
It can be run with a `drush` command or alternatively just set up the old fashioned way as a straight cron job. See https://www.drupal.org/docs/7/setting-up-cron-for-drupal/configuring-cron-jobs-using-the-cron-command